scipy/special/cephes/lanczos.h

```diff
@@ -130,4 +130,6 @@ static const double lanczos_sum_near_2_d[12] = {
 
 static const double lanczos_g = 6.024680040776729583740234375;
 
+double lanczos_sum_expg_scaled(double x);
+
 #endif
```

Comment:
You should replace the include guards with #pragma once!

Reply:
Ha. Amazed I missed that! Will do.

Reply:
Wait. That's just the header in the original cephes library. It's not clear to me why I even touched it. The one in the new C++ library is https://github.com/scipy/scipy/pull/19954/files#diff-2e3af56b6eca119873a12e189a6d166ccfce02406b414a5f099542163cc117b4

Reply:
@izaid. Good catch. That was just a file I'd inadvertently changed. I shouldn't have even touched it.